src/plugins/expressions/public/expression_renderer.tsx

```diff
@@ -135,15 +159,9 @@ export const ExpressionRendererImplementation = ({
 
   return (
     <div {...dataAttrs} className={classes}>
-      {state.isEmpty ? <EuiLoadingChart mono size="l" /> : null}
-      {state.isLoading ? <EuiProgress size="xs" color="accent" position="absolute" /> : null}
-      {!state.isLoading && state.error ? (
-        renderError ? (
-          renderError(state.error.message)
-        ) : (
-          <div data-test-subj="expression-renderer-error">{state.error.message}</div>
```

Comment:
1. Default error state now fallbacks to default of `render.ts` which is toast from notification service
2. If user of this react component provided `renderError`, then the component itself will handle the error and render error component returned from renderError()